contracts/scripts/native_solc_compile_all

```diff
@@ -81,6 +81,7 @@ $SCRIPTPATH/native_solc8_6_compile vrf/BatchBlockhashStore.sol
 $SCRIPTPATH/native_solc8_6_compile vrf/BatchVRFCoordinatorV2.sol
 $SCRIPTPATH/native_solc8_6_compile vrf/testhelpers/VRFCoordinatorV2TestHelper.sol
 $SCRIPTPATH/native_solc8_6_compile vrf/VRFCoordinatorV2.sol 10000
+$SCRIPTPATH/native_solc8_6_compile dev/vrf/VRFOwner.sol
```

Comment:
given that this was audited, does this belong in the main `vrf` folder instead of `dev` ?

Reply:
Yup can move it out.